### DISCUSSION TITLE
Steper Gear Ratio

### DISCUSSION THREAD

**Discussion by Leygan1** [kachurovskiy/nanoels#193] [Q&A]
Hello again. So finally, I am finishing the build that I have been planning for almost 2 years. I am trying to install it on this old Halifax Lathe.

![image](https://github.com/kachurovskiy/nanoels/assets/90058128/98b53370-2b02-4206-8214-79f8047ab150)

I have an H2 and I'm trying to use a NEMA34 86BYGH2120-6004A motor, which is 8.4Nm and has 200 steps (quite a big boy). I can't find the correct belt for the gear that it has, so I am opting for direct drive. Right now, I have printed the gear and it has a quite bad ratio of 2.13333:1. I will try to adjust it to 2:1, but will this work anyway?

200 * 2.13333 = 426.6666. I will write the code for 426 steps or with another gear 400 steps. But the motor is 200 steps, what will happen?

edit:

Okay, I need more info about stepper motors... I don't understand what I am doing :D

**Comment by kachurovskiy** [comment]
You can keep motor steps at 200 and simply adjust the LEAD_SCREW_DU to indicate how far your carriage travels for 1 turn of the motor.
